src/components/Prices/index.tsx

```diff
@@ -31,11 +34,21 @@ interface PricesProps {
 }
 
 const Prices = ({ priceLevels }: PricesProps) => (
-  <PriceContainer>
-    {priceLevels.map((level) => (
-      <PriceItem key={level.id} data={level} />
-    ))}
-  </PriceContainer>
+  <>
+    <PriceCarousel>
```

Comment:
Would prefer to name this "PriceCarouselMobile" so that it is clear that this is mobile only.